src/App.js

```diff
@@ -1,20 +1,65 @@
-import { FaLinkedin } from 'react-icons/fa';
-import './global.css';
+import "./global.css";
+import Logo from "./assets/HotSoupLogo .png";
+import PhonePreview from "./assets/PhonePreview.png";
+import PhoneSearch from "./assets/PhoneSearch.png";
 
+import React from "react";
 
-function App() {
-  return (
-    <>
-      <section className='text'>
-        <h1> Website under construction; will be coming soon, thank you! </h1>
-        <h2> Check us on out on our social media page </h2>
-        <div className='socials'>
-          <a href='https://www.linkedin.com/company/hotsoup-info/' target='_blank' rel='noreferrer'><FaLinkedin/></a>
-        </div>
-      </section>
-
-    </>
-  );
-}
+const App = () => {
+	return (
+		<div className="body">
+			<nav className="navbar">
+				<img src={Logo} alt="hotsouplogo" />
+				<h2>HotSoup</h2>
+			</nav>
+			<div className="main">
+				<div className="main-text">
+					<h1>COMING SOON</h1>
+					<p>
+						HotSoup is an opensource mobile app project, which is coming soon to
+						a mobile app platform near you! It was started to help those who
+						suffer from food insecurity find their next meal. All technology and
+						engineering is 100% volunteer based, helping seasoned engineering
+						sharpen their skills, new devs get some real-world experience (to
+						help land that first job that always seems to require experience),
+						all while using their skills to give back to the communities. <br />{" "}
```

Comment:
Please remove breaks and use other forms of code to give it the styling that you prefer.

Reply:
Remove breaks and made paragraphs